### PR TITLE
Fixing HTML and CSS

### PR DESCRIPTION
**Pull request by DHCJS**
This is the change I've made. 
image
and this too
image
and this one too
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,33 +30,33 @@
...
         <h4 class="hero-subtitle">Learn work in a development team by making small contributions</h4>
         <a class="hero-button" href="https://github.com/jeffreycharters/station" target="_blank">Get started!
           &raquo;</a>
       </div>
     </section>
 
     <section class="about">
-      <h1 id="h1">station</h1>
-      <p>Hi. This is a community effort. &#127757;</p>
+      <h1 id="h1">Station</h1>
```
Comment (DHCJS) [inline index.html:41]: Change the into Upper case

File: index.html [hunk 2]
```diff
@@ -30,33 +30,33 @@
   <div class="container">
     <section class="hero-section text-center">
       <div class="hero-content">
-        <h1 class="hero-title">Discover the joy of open source!</h1>
+        <h1 class="hero-title">Discover The Joy of Open Source!</h1>
```
Comment (DHCJS) [inline index.html:33]: Change into Upper case

File: index.html [hunk 3]
```diff
@@ -30,33 +30,33 @@
...
         <a class="hero-button" href="https://github.com/jeffreycharters/station" target="_blank">Get started!
           &raquo;</a>
       </div>
     </section>
 
     <section class="about">
-      <h1 id="h1">station</h1>
-      <p>Hi. This is a community effort. &#127757;</p>
+      <h1 id="h1">Station</h1>
+      <p>Hi, this is a community effort. &#127757;</p>
```
Comment (DHCJS) [inline index.html:42]: Coma instead of dot

File: index.html [hunk 4]
```diff
@@ -30,33 +30,33 @@
...
       <p>We're curious to see where this takes us. &#128513;</p>
       <!-- motivational image -->
       <iframe src="https://giphy.com/embed/Ju7l5y9osyymQ" frameBorder="0" id="embed"></iframe>
       <!-- image attribution -->
       <p><a href="https://giphy.com/">via GIPHY</a></p>
       <p id="embed"><a class="link" href="https://giphy.com/" target="_blank">via GIPHY</a></p>
-      <p>Check us out on <a class="link" href="https://github.com/jeffreycharters/station" target="_blank">Github</a>!
+      <p>Check us out on <a class="link" href="https://github.com/jeffreycharters/station" target="_blank">GitHub</a>!
```
Comment (DHCJS) [inline index.html:49]: Fixing H to uppercase

File: index.html [hunk 5]
```diff
@@ -30,33 +30,33 @@
...
       </p>
       <p>Join us on the <a class="link" href="https://discord.gg/NhAfhBnh8g" target="_blank">Self Taught Tech Network
-          Discord server</a> &#128072</p>
+          Discord Server</a> &#128072</p>
```
Comment (DHCJS) [inline index.html:52]: S uppercase

File: index.html [hunk 6]
```diff
@@ -86,7 +86,7 @@ <h2>Helpful Links </h2>
       </div>
     </section>
 
-    <section class="testinomials">
+    <section class="testimonials">
```
Comment (DHCJS) [inline index.html:89]: fixing typo

File: index.html [hunk 7]
```diff
@@ -30,33 +30,33 @@
...
-          Discord server</a> &#128072</p>
+          Discord Server</a> &#128072</p>
     </section>
 
     <section class="links">
       <h2>Helpful Links </h2>
-      <p>check here to know <a class="link" href="https://www.koskila.net/how-to-git-merge-upstream/"
-          target="_blank">how to git merge upstream🧑‍💻</a></p>
-      <p>click here for <a class="link"
+      <p>Check here to know <a class="link" href="https://www.koskila.net/how-to-git-merge-upstream/"
```
Comment (DHCJS) [inline index.html:57]: fixing lower case into uppercase

File: style.css [hunk 8]
```diff
@@ -129,6 +129,7 @@ h2 {
   justify-content: space-around;
   border: 3px solid #de51ff;
   padding: 1em;
+  border-radius: 20px;
```
Comment (DHCJS) [inline style.css:132]: Round Edges

**Review by DHCJS — COMMENTED** [review]
Fixing Typo, Changing Text and border-radius

**Review by michaella23 — APPROVED** [review]
Appreciate your attention to detail!

**Pull request merged**